client/src/js/selectors/index.js

```diff
@@ -27,10 +27,15 @@ export const getVisibleArtifacts = createSelector(
     switch (allFilter) {
       case true:
         return allArtifacts.filter(artifact => 
-          artifact.title
+          (artifact.title
               .toLowerCase()
               .trim()
-              .includes(artifactFilter.toLowerCase().trim()) || '^\\s*$'.match(artifactFilter)
+              .includes(artifactFilter.toLowerCase().trim()) || '^\\s*$'.match(artifactFilter))
+          ||
+          (artifact.tags
```

Comment:
This is a list of items -> the filtering on Artifact titles worked because it was an array of chars. This is an array of arrays of chars.

Reply:
> > It doesn't work atm, please see comment about filtering -> You are attempting to filter a list of lists the same way a list is filtered
> 
> Are you sure about this? Because it works for me...
> 
> ![image](https://user-images.githubusercontent.com/49027863/66979124-2122df80-f0f8-11e9-9fd8-44af799a00ea.png)

@Dezyh can you double check, I'll repull it and check.

Reply:
It seems to work fine for me...